deployer/src/main/java/eu/seaclouds/policy/SeaCloudsInitializerPolicy.java

```diff
@@ -0,0 +1,544 @@
+                            "                                    \"column\": \"value\"," +
+                            "                                    \"function\": \"mean\"," +
+                            "                                    \"query\": \"select mean(value) from \\\"" + metricName + "\\\" where $timeFilter group by time($interval) order asc\"," +
+                            "                                    \"refId\": \"A\"," +
+                            "                                    \"series\": \"" + metricName + "\"" +
+                            "                               }]," +
+                            "                  \"timeFrom\": null," +
+                            "                  \"timeShift\": null," +
+                            "                  \"title\": \"" + metricName + "\"," +
+                            "                  \"tooltip\": { \"shared\": true, \"value_type\": \"cumulative\"}," +
+                            "                  \"type\": \"graph\"," +
+                            "                  \"x-axis\": true," +
+                            "                  \"y-axis\": true," +
+                            "                  \"y_formats\": [\"short\",\"short\"]" +
+                            "              }]," +
+                            "      \"title\": \"" + entity.getDisplayName() + "\"" +
+                            "},";
+            rowOptions += currentRow;
+        }
+
+        // Remove last ,
+        rowOptions = rowOptions.substring(0, rowOptions.length() - 1);
+        grafanaInitializerPayload = globalGrafanaOptionsJSON + rowOptions + "] } }";
+
+    }
+
+    @Override
+    public void setEntity(EntityLocal entity) {
+        super.setEntity(entity);
+
+        // SeaCloudsInitializerPolicy should only be attached to Application
+        if (!entity.getApplication().equals(entity)) {
+            //TODO: Check if this exception stops the deployment process
+            throw new RuntimeException("SeaCloudsInitializerPolicy must be attached to an application");
+        }
+
+        parseMonitoringRules();
+        parseAgreements();
+        setupObserverInitializerPayload();
+        setupGrafanaInitializerPayload();
+        // Overriding the default START/STOP behaivour to include the aditional tasks required by
+        // SeaClouds while keeping the original behaviour. It allows to initialize and clean-up SeaClouds
+        // components when an application is added/removed.
+        ((EntityInternal) entity).getMutableEntityType().addSensor(SLA_ID);
+        ((EntityInternal) entity).getMutableEntityType().addSensor(T4C_IDS);
+        ((EntityInternal) entity).getMutableEntityType().addEffector(newStartEffector());
+        ((EntityInternal) entity).getMutableEntityType().addEffector(newStopEffector());
+
+    }
+
+    private Effector<Void> newStartEffector() {
```

Comment:
I think you don't need to define `Effector`s here, it should be enough to add

```
   private SensorEventListener<Lifecycle> eventHandler = new SensorEventListener<Lifecycle>() {
        @Override
        public void onEvent(SensorEvent<Lifecycle> event) {
            //Start the backup only once the entity is in running state.
            //This will ensure that we have the IP of the machine to
            //connect to.
            if (event.getValue() == Lifecycle.RUNNING) {
                installSlaAgreement();
                installMonitoringRules();
                installInfluxDbObservers();
                installGrafanaDashboards();
            }
        }
    };
```